app/src/main/java/org/akvo/flow/service/DataSyncService.java

```diff
@@ -591,7 +592,13 @@ private void checkDeviceNotifications() {
                 Timber.e("Could not retrieve missing files");
             }
         } catch (Exception e) {
```

Comment:
Small comment here... maybe you could have used instead
```java
try {
} catch (HttpException e) {
} catch (Exception e) {
}
```

Instead of having the `if()` block